### DISCUSSION TITLE
官网是如何维护，有仓库地址吗？

### DISCUSSION THREAD

**Discussion by David-Liu5460** [ant-design/ant-design#51183] [Q&A] [answered]
他的官网组件文档是怎么更新的，markdown更新之后是如何更新到官网的？这个有什么设计吗

**Comment by zombieJ** [comment, marked as answer]
就是这个 repo。你 clone 下来 npm start 就能看到了。发布版本的时候会更新到 github pages，官网就更新了。

**Reply by David-Liu5460** [reply]
markdown转html在这个repo没有看到呀，另外这个markdown还需要解析code的src路径吧，这一步是在哪里完成的的呀
![image](https://github.com/user-attachments/assets/47ca99da-b53a-4d7d-906d-585b11dfd9c3)

**Reply by afc163** [reply]
源码都在这里了，欢迎自由探索

**Reply by David-Liu5460** [reply]
> 源码都在这里了，欢迎自由探索

好的，我研究一下再和大佬交流哈哈～～～

**Reply by David-Liu5460** [reply]
组件文档的维护为什么不用storybook？

**Comment by David-Liu5460** [comment]
npm start看到页面了，我再看一下怎么做的哈，thx～～～
